pkg/manager/member/utils.go

```diff
@@ -532,3 +540,36 @@ func GetPVCSelectorForPod(controller runtime.Object, memberType v1alpha1.MemberT
 	}
 	return l.Selector()
 }
+
+// ShouldMoveTiKVLogPath returns if rocksdb log path move from db/LOG to rocksdb.info and raft log move from raft/LOG to raftdb.info
+func ShouldMoveTiKVLogPath(image string) bool {
+	_, version := parseImage(image)
+	v, err := semver.NewVersion(version)
+	if err != nil {
+		klog.Errorf("Parse version %s failure, error: %v", version, err)
+		return false
+	}
+	if tikvLessThanV500.Check(v) {
+		return true
+	}
+	return false
+}
+
+// parseImage returns the image name and the tag from the input image string
+func parseImage(image string) (string, string) {
+	var name, tag string
+	colonIdx := strings.LastIndexByte(image, ':')
+	if colonIdx >= 0 {
+		name = image[:colonIdx]
+		tag = image[colonIdx+1:]
+	} else {
+		name = image
+	}
+
+	// check dirty version like: v5.1.1-20210926
```

Comment:
No need to separate the tag here, right?
1. This is the real tag
2. It works with L552.

Reply:
Yes, retest and works well.😂